README.md

```diff
@@ -3,6 +3,8 @@ Heureka 'Ověřeno zákazníky' PHP API
 
 [Heureka Ověřeno zákazníky](http://overeno.heureka.cz/) (ShopCertification) service API client implementation for PHP.
 
+Note: For **non-PHP** implementations check the raw API documentation in the `docs` folder.
```

Comment:
raw bych odstranil ... je to prostě API dokumentace. Jo a můžeš tam udělat klidně link na ty soubory. Něco jako "... API documentation available in both *czech*(link) and *english*(link)"